src/client/ducks/pullrequests/queries/pullRequestFile.graphql

```diff
@@ -0,0 +1,24 @@
+query pullRequestFiles($id: Int!, $name: String) {
```

Comment:
yes, I'll fix it, but the query name does not affect anything just could confuse if you try to explore requests in browser :)